elytron/src/main/java/org/wildfly/extension/elytron/ExpressionResolverResourceDefinition.java

```diff
@@ -62,7 +62,7 @@
  */
 class ExpressionResolverResourceDefinition extends SimpleResourceDefinition {
 
-    public static final Pattern INITIAL_PATTERN = Pattern.compile("\\$\\{.+::.*:.+}");
+    public static final Pattern INITIAL_PATTERN = Pattern.compile("\\$\\{.+::.+}");;
```

Comment:
There are two semicolon characters ending the `Pattern.compile()`

Reply:
@michpetrov Are you able to provide some more details on what the issue was and about this fix?

Reply:
@fjuma in the encrypted expression format `${ENC:resolver-name:<encoded-string>}` the "resolver-name" is supposed to be optional, but the INITIAL_PATTERN only accepts expression strings with the resolver name included. The issue only comes up when a user tries to put encrypted expressions in system properties.